CONTRIBUTING.md

```diff
@@ -39,6 +39,38 @@ request into the body of the commit message.
 
 If the Travis tests are failing on your branch, you should look at the logs to figure out why. Sometimes they fail spuriously, in which case you can post a comment requesting that a project owner kick the build.
 
+# Error handling
+
+All errors must be addressed in some way: That may be simply by returning an
+error up the stack, or by handling it in some intelligent way where it is
+generated, or explicitly ignored by assigning it to `_`. We use the `errcheck`
```

Comment:
Nit: "or explicitly ignored by assinging it to `_`" -> "or by explicitly ignoring it and assigning to `_`"?